packages/ruby.rb

```diff
@@ -56,7 +62,7 @@ def self.build
   def self.check
     # Do not run checks if rebuilding current ruby version.
     # RUBY_VERSION is a built-in ruby constant.
-    system 'make check || true' unless RUBY_VERSION == '3.2.0'
+    system 'make check || true' unless RUBY_VERSION == '3.2.2'
```

Comment:
`RUBY_VERSION == version`?

Reply:
Actually, why do we even need this conditional?

Reply:
Change made. Conditional is to reduce the time for rebuilds with minor changes (such as happened with this build...)